### PR TITLE
Template clean up

### PR DESCRIPTION
**Pull request by trickreich**

### CODE REVIEW

File: src/AppBundle/Resources/views/templates/artist-detail.html.twig [hunk 1]
```diff
@@ -7,8 +7,61 @@
...
+                    'blocks' : content.blocks
+                } %}
+            </div>
+
+            <div class="grid__item desk--four-twelfths">
+                <div class="facts">
+                    <div class="facts-header">Facts</div>
+                    <div class="facts-content">
+                        <p>
+                            {% if content.genre|default() %}
```
Comment (danrot) [inline src/AppBundle/Resources/views/templates/artist-detail.html.twig]: Wouldn't `content.genre is not empty` also work and describe a little bit better what's happening?

Comment (danrot) [inline src/AppBundle/Resources/views/templates/artist-detail.html.twig]: Nevermind, just saw that it has been done everywhere like that, probably not worth changing it.

File: src/AppBundle/Resources/views/templates/blog.html.twig [hunk 2]
```diff
@@ -11,8 +11,8 @@
             <div class="blog-item">
                 <div class="grid">
                     <div class="grid__item desk--five-tenths lap--one-half palm--one-whole">
-                        {% if article.image|length and article.image|first.thumbnails['600x300'] is defined %}
-                                {% set image = article.image|first %}
+                        {% if article.excerpt.images|length and article.excerpt.images|first.thumbnails['600x300'] is defined %}
```
Comment (danrot) [inline src/AppBundle/Resources/views/templates/blog.html.twig]: Does the `article.image` still exist? If yes I would probably use it as a fallback here. But that's something that's not possible with the `ArticleBundle`, right?

Comment (trickreich) [inline src/AppBundle/Resources/views/templates/blog.html.twig]: That's right, it's not possible at the moment. We do not have any content of the ArticleDocument here.

File: src/AppBundle/Resources/views/templates/artist-detail.html.twig [hunk 3]
```diff
@@ -7,8 +7,61 @@
...
+        <div class="grid content">
+            <div class="grid__item desk--eight-twelfths">
+                {% include ('AppBundle:renderer:block-renderer.html.twig') with {
+                    'blocks' : content.blocks
+                } %}
+            </div>
+
+            <div class="grid__item desk--four-twelfths">
+                <div class="facts">
+                    <div class="facts-header">Facts</div>
```
Comment (danrot) [inline src/AppBundle/Resources/views/templates/artist-detail.html.twig]: Displayed in wrong font?

File: src/AppBundle/Resources/views/templates/artist-detail.html.twig [hunk 4]
```diff
@@ -7,8 +7,61 @@
...
+                    <div class="facts-content">
```
Comment (danrot) [inline src/AppBundle/Resources/views/templates/artist-detail.html.twig]: It looks to me like everything you've added here is not working in the preview.

**Pull request merged**